scripts/build.mjs

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env zx
+import { $, path, os, fs, argv, fetch, cd, within, chalk } from 'zx'
+import { finished } from 'node:stream/promises'
+import archiver from 'archiver'
+
+if (argv.h || argv.help) {
+  console.log(`${argv._} [--target <name of target>] [--node-version <version>]`)
+  console.log()
+  console.log('Targets can be "linux", "linux-amd64", "linux-arm64", "darwin", "darwin-amd64", "darwin-arm64".')
+  console.log('If target does not indicate an architecture it will build all architectures for that platform.')
+  console.log('Target may be specified multiple times to build multiple targets.')
+  console.log()
+  console.log('Node version should be in the format "v#.#.#". For example: "v18.4.0')
+  console.log()
+  process.exit(0)
+}
+
+const targets = (Array.isArray(argv.target) ? argv.target : [argv.target])
+  .filter(v => !!v)
+  .map(v => v.toLowerCase())
+
+const nodeVersion = argv.nodeVersion || process.version
+const matrix = [
+  // [platform, arch, download extension, target extension, mirror]
+  ['linux', 'amd64', '.tar.xz', '', ''],
```

Comment:
I'm now using `amd64` instead of `x64` and just mapping that to `x64` when downloading node and finding the pre-built caxa stub file

Reply:
Good work!